docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+    });
+
+    module.exports = todoActions;
+```
+
+## todoStore.js
+`Tux` provides all of the glue code needed to build stores and register them with the `TuxActions` dispatcher. Now that we have created our actions and views. Let's create our store for our Todos that respond to our actions and provide the data for our views. `Tux` here again significantly reduces the boilerplate required to get a store up and running.
+
+```javascript
+    var ActionStores = require('tux/Stores/ActionStores');
+
+    var todoStore = ActionStores.createStore({
+      _todos: [],
+
+      getAll: function () {
+        return this._todos;
+      },
+
+      onAdd: function (todo) {
+        this._todos.push({
+          text: todo.text,
+          id: this._todos.length
+        });
+        this.emitChange();
+      },
+
+      onRemove: function (todo) {
+        this._todos.splice(todo.id, 1);
+        this.emitChange();
+      },
+
+      register: function () {
+        return {
+          todos: {
+            add: this.onAdd,
+            remove: this.onRemove
+          }
+        };
+      }
+    });
+
+    module.exports = todoStore;
+```
+
+## TuxTodoApp.js and index.html
+Finally after putting together all of the pieces, let's render the `TodoViewOwner` component to the DOM.
+
+```javascript
+    var React = require('tux/React');
+    var TodoViewOwner = require('./TodoViewOwner.jsx');
+
+    // Render the TodoViewOwner component to the element with the id 'main'
+    React.render(<TodoViewOwner />, document.getElementById('main'));
+```
+
+```markup
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <title>TuxTodoApp</title>
+    </head>
+    <body>
+      <div id="main"></div>
+      <script src="bundle.js"></script>
+    </body>
+```
+
+## Bundling and serving
+Once we have all of these pieces we can run the `npm start` command to get our JavaScript to bundle together. Following that we can either start a simple HTTP server or we can open the `index.html` page in our browser of choice. Now you should be able to add and delete Todos from your basic `TuxTodoApp`!
+
+## Conclusion
+This example should illustrate the versatility and semantic nature of `Tux`. Each of the opinionated components provides powerful convenience methods to allow you to do a multitude of operations. For one you have the ability to share properties and methods between components without using the standard `React` pass down syntax. Another benefit is the ability to harness the performance benefits of the Mutable class components, where it is determined which components need to be re-rendered in the virtual DOM. Then there is the ease in which you can declare your `Flux` actions, create your stores with reduced boilerplate, and finally the ability to register your stores with your Actions.
```

Comment:
Change 'to allow' to 'that allow' in second sentence.

Change 'another benefit is' to 'You have'. Remove ...where it is determined which components need to be re-rendered in the virtual DOM.

Change 'and finally the ability to' to 'and register'